File: CS-Examples/04_RowsColumns/UnhideRowsAndColumns.cs
using System;
using System.Windows.Forms;
using Spire.Xls;

namespace UnhideRowsAndColumns
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

		private void btnRun_Click(object sender, EventArgs e)
		{
            //Create a workbook
			Workbook workbook = new Workbook();

            //Load the document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\CommonTemplate1.xlsx");

            //Get the first worksheet
            Worksheet sheet = workbook.Worksheets[0];

            //Unhide the row
            sheet.ShowRow(15);

            //Unhide the column
            sheet.ShowColumn(4);

            //Save the document
            string output = "UnhideRowsAndColumns.xlsx";
			workbook.SaveToFile(output, ExcelVersion.Version2013);

            //Launch the file
			ExcelDocViewer(output);
		}
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}

File: VB-Examples/04_RowsColumns/UnhideRowsAndColumns.vb
Imports Spire.Xls

Namespace UnhideRowsAndColumns

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            'Create a new instance of the Workbook class.
            Dim workbook As New Workbook()
            'Load the Excel document from the specified path.
            workbook.LoadFromFile("..\..\..\..\..\..\Data\HideRowsAndColumns.xls")
            'Retrieve the first worksheet from the workbook.
            Dim worksheet As Worksheet = workbook.Worksheets(0)
            'Hide the second column of the worksheet.
            worksheet.HideColumn(2)
            'Hide the fourth row of the worksheet.
            worksheet.HideRow(4)
            ' Specify the file name
            Dim output As String = "HideRowsAndColumns.xlsx"
            'Save the modified workbook to a file with the specified name and Excel version.
            workbook.SaveToFile(output, ExcelVersion.Version2010)
            ' Release the resources used by the workbook
            workbook.Dispose()

            'Launch the file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
